packages/ketchup/src/managers/kup-dates/kup-dates.ts

```diff
@@ -798,4 +799,25 @@ export class KupDates {
     };
 
     formatToLocaleSimple = (date: Date) => dayjs(date).format('DD/MM/YYYY');
+
```

Comment:
Since new methods of the `KupDates` manager were added, they must also be added in the showcase page of `KupDates`.